src/dmclock/test/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+INCLUDE (CheckIncludeFiles)
```

Comment:
I see.... unexpected.

I'm left wondering why this is a subtree when other dependencies are *submodules*? What's the advantage a subtree has over a submodule?

I'll submit this to the dmclock repo.